### PR TITLE
Improve !help message

### PR DESCRIPTION
**Pull request by aidalgol**
Add the missing server argument to the help message for !whois, and make this argument consistent across all command help messages.

(Related: #547)

### CODE REVIEW

File: lib/bridge/MatrixHandler.js [hunk 1]
```diff
@@ -203,33 +203,34 @@ MatrixHandler.prototype._onAdminMessage = Promise.coroutine(function*(req, event
...
             "!quit": {
                 example: `!quit`,
-                summary: `Leave all bridged channels and remove your connection to IRC`,
+                summary: "Leave all bridged channels, on all networks, and remove your " +
+                         "connections to all networks.",
             },
             "!cmd": {
-                example: `!cmd [irc.server] COMMAND [arg0 [arg1 [...]]]`,
+                example: `!cmd [irc.example.net] COMMAND [arg0 [arg1 [...]]]`,
                 summary: `Issue a raw IRC command. These will not produce a reply.`,
```
Comment (Mikaela) [inline lib/bridge/MatrixHandler.js]: Could you also say that the COMMAND has to be capitalized regardless of IRC generally being a case-insensitive protocol?

Comment (aidalgol) [reply lib/bridge/MatrixHandler.js]: Done.  I have created an issue for making `!cmd` case insensitive.

**Comment by aidalgol** [comment]
I can't find the failure in the Travis CI report, and `npm run check` passes on my local system.

**Comment by Half-Shot** [comment]
LGTM 👍

**Pull request merged**